apps/bfd-server/dev/api-changelog.md

```diff
@@ -704,3 +704,7 @@ Future updates may add `Coding.display` values for additional fields.
 	* The "FIXME this should be mapped as an extension valueIdentifier instead of as a valueCodeableConcept" issues were addressed by creating a new common method for adding identifiers to an extension instead of a codeable concept for these fields. The new method is called addExtensionValueIdentifier in TransformerUtils.
 	* The "FIXME: check if this field is non-nullable and if not remove the 'if' check" issues were addressed by comparing the fields to their definition in the rif-layout-and-fhir-mapping.xlsx file. Most fields were found to be non-nullable and so the "if" check was removed.
  
+ ## BFD-874 Fix diagnosis.sequence in V1 and V2
+
+ * Following FHIR mapping changes were made:
+  * Adds diagnosis.sequence if present in EOB.diagnosis, when a DRG is reported, for V1 & V2
```

Comment:
Change to "Now, when a DRG is reported for V1 and V2, diagnosis.sequence is present in EOB.diagnosis. Previously, diagnosis.sequence was not present"

Reply:
domne